### ISSUE TITLE
vagrant install guide errors

### ISSUE THREAD

**Issue by danehans** [gemini-project/gemini#10]
The install guide contains the following errors:

Diagram reference is missing: https://github.com/stephenrlouie/gemini_images/blob/master/high-level.png

Move demo to a section near the bottom and add a reference to it. Although most dev's will not use the demo, but I think it's still important to have.

Create a table of contents for items in Overview to the associated item:
Ex>
```
##### Overview
[Headers](#headers)  
[Emphasis](#emphasis)  
...snip...    
<a name="headers"/>
## Headers
```

A good reference: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Change Pull the cent image a to Pull the centos image. (specify the os details such as version).

**Comment by stephenrlouie** [comment]
We have to pull the cent image because that is the build image, I'll update the name so its cent6.7, We don't pull the coreos image, we only fetch the vmlinuz, and PXE files.

**Comment by danehans** [comment]
@stephenrlouie as you can see, I merged #10. Please review the vagrant readme. Looks like the diagram is still missing. Please add the diagram or remove the section. Thanks!

**Comment by stephenrlouie** [comment]
@danehans Can we close this?
